db/src/main/resources/db/migration/V1_040__add_qualifying_benefit_status_to_payment_cycle.sql

```diff
@@ -0,0 +1 @@
+alter table payment_cycle add column qualifying_benefit_status varchar;
```

Comment:
What are you going to do for those claims already in the Production database?

Reply:
Fair point. I'll add an update statement.